sdcm/tester.py

```diff
@@ -262,6 +262,11 @@ def get_cluster_gce(self, loader_info, db_info, monitor_info):
             loader_info['n_local_ssd'] = self.params.get('gce_n_local_ssd_disk_loader')
         if db_info['n_nodes'] is None:
             db_info['n_nodes'] = self.params.get('n_db_nodes')
+        if db_info['n_nodes_dcs'] is None:
+            db_info['n_nodes_dcs'] = self.params.get('n_db_nodes_dcs')
+            db_info['n_nodes'] = []
+            for i in self.params.get('n_db_nodes_dcs').split():
```

Comment:
db_info['n_nodes_dcs'] can be used here instead of getting the param twice, and may be It's worth checking it's not None